manager/orchestrator/task_test.go

```diff
@@ -105,3 +109,56 @@ func TestIsTaskDirtyPlacementConstraintsOnly(t *testing.T) {
 	service.Spec.Task.Placement.Constraints = nil
 	assert.False(t, IsTaskDirtyPlacementConstraintsOnly(service.Spec.Task, task))
 }
+
+// Test Task sorting, which is currently based on
+// Status.AppliedAt, and then on Status.Timestamp.
+func TestTaskSort(t *testing.T) {
+	var tasks []*api.Task
+	size := 5
+	seconds := int64(size)
+	for i := 0; i < size; i++ {
+		time.Sleep(1)
+		task := &api.Task{
+			ID: "id_" + strconv.Itoa(i),
+			Spec: api.TaskSpec{
```

Comment:
Actually; looks like the `Spec` isn't needed for this test to work; better to remove it